config/locales/js-de.yml

```diff
@@ -386,9 +386,9 @@ de:
       delete: Beziehung löschen
     inplace:
       button_edit: "%{attribute} bearbeiten"
-      button_save: "Speichern"
-      button_save_and_send: "Speichern mit E-Mail-Benachrichtigung"
-      button_cancel: "Abbrechen"
+      button_save: "%{attribute} Speichern"
```

Comment:
@0xF013 sorry for just noticing this. To match with the English variant, I guess there should be a colon after the attribute name.